### PR TITLE
Added Favorites Functionality

### LINKED ISSUE #38: Add items to favorite
It could be great to list in local storage your favorite items.
It can be a new jumbotron under the first one on the homepage.
A jumbotron called "favorite". 

To add an item to favorite, there will be a little heart in the top right corner.
When the user press this heart icon, the album/playlist will be added to favorites.

Genre & mood card will be treated in another story

### PR DESCRIPTION
**Pull request by adhammo**
Resolves #38.

I added a "love" button to add a playlist to favorites which will add it to local storage and also added a jumbotron to display favorite playlists.

image

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -38,4 +47,23 @@ export class HomeComponent implements OnInit {
     window.location.href = item.uri;
   }
 
+  isFavorite(id): boolean {
+    return this.favorites.some((fav) => fav == id);
+  }
+  
+  favorite(id): void {
+    if (this.isFavorite(id))
+    {
```
Comment (arnaud512) [inline src/app/home/home.component.ts:56]: brackets on the same line please :)

**Review by arnaud512 — CHANGES_REQUESTED** [review]

**Comment by arnaud512** [comment]
Wow really nice looking!! I really like it!
Can you add this feature inside genres/moods items? (ex: http://localhost:4200/playlists/toplists?name=Top%20Lists)
I will merge it once done!

**Comment by adhammo** [comment]
Hey, @arnaud512.

I added favorites functionality to the genres and playlists.

**Comment by adhammo** [comment]
@arnaud512, can you review the changes?

**Comment by arnaud512** [comment]
Oups I was not as precise as I thought! 
I would like a unique place in the app to get all my favorite items.
So that place at home is where you will see all your favorites items! (from home and from genres/moods)
It that correct for you?

**Comment by arnaud512** [comment]
But after that, it will be perfect for me!

**Comment by adhammo** [comment]
I added the favorites page

**Comment by arnaud512** [comment]
Ahh!! no! not a favorite page!
A unique jumbotron called "favorite" at home. Just that!
That gets all your favorites from all the app.
No need to distinguish from where they are coming!

**Pull request closed without merging**